src/linux/agent/system-server.js

```diff
@@ -26,7 +26,7 @@ function init() {
 }
 
 rpc.exports = {
-  enumerateApplications: function () {
+  enumerateApplications: function (userid=0) {
```

Comment:
Should only use ES5 syntax here so Duktape can load this agent in case Frida is built without V8.

Reply:
addressed by 5fbd9a441df529369caa3bac55fead2cf65d1892